content/getting-started/adding-apps-from-custom-sources.md

```diff
@@ -23,8 +23,8 @@ To add an app from custom source:
 
     {{< figure size="" src="../uploads/add-app-from-custom-source2.png" caption="" >}}
 
-    * Enter the **Repository URL** for cloning the repository. The URL should be in the following format:
-    `https://example.com/username/repo.git` or `git@example.com/username/repo.git`. 
+    * Enter the **Repository URL** for **cloning** the repository. You can find the URL from your repository on clicking **Clone** / **Clone or download**. Depending on the authentication method, choose either the HTTPS or SSH URL. The most common URL formats are as follows:
```

Comment:
Currently, we do not support http auth for https:// repos, so 
> Depending on the authentication method
seems to be a bit misleading. Perhaps replace that sentence with something along the lines of:
> Depending on whether you're accessing a public or private repo

Reply:
Amended as you suggested.